website/src/css/_showcase-section.scss

```diff
@@ -0,0 +1,55 @@
+// Showcase buttons
+
+@mixin showcase-default-btn {
+  background: radial-gradient(103.29deg, #7633e6 -14.83%, #5d71e6 102.26%);
```

Comment:
I think Asaf meant that you could have moved it entirely somewhere under Showcase component.

Reply:
This entire file, decided to remove it as it was not looking to be adding any useful changes.